src/utils/DiscordUtil.ts

```diff
@@ -0,0 +1,31 @@
+import { Guild, GuildMember } from "discord.js";
+
+class DiscordUtil {
+	static async getGuildMember(value: string, guild: Guild): Promise<GuildMember | undefined> {
+		if (value === "") return;
+
+		// UserID
+		if ((/^[0-9]+$/g).test(value)) {
+			try {
+				return await guild.members.fetch(value);
+			} catch (error) {
```

Comment:
You can remove `(error)` if you're not using it.

Reply:
`Parameter name expected`

Reply:
You should also remove ()